src/Label.js

```diff
@@ -1,24 +1,30 @@
 import styled from 'styled-components'
-import { space, fontSize, color, propTypes } from 'styled-system'
+import { space, fontSize, fontWeight, color, propTypes } from 'styled-system'
 import theme from './theme'
```

Comment:
Don't see the need for importing theme here as well.

Reply:
Importing the `theme` here is what @jes708 added so that he could add the theme as a default prop.